test/DynamoCoreWpfTests/DynamoViewModelUnitTest.cs

```diff
@@ -36,6 +37,16 @@ protected override DynamoModel GetModel()
             return ViewModel.Model;
         }
 
+        protected static string GetAppDataFolder()
+        {
+            var folder = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
+            var dynamoVersion = FileVersionInfo.GetVersionInfo(Assembly.GetExecutingAssembly().Location);
+            var appDataFolder = Path.Combine(Path.Combine(folder, "Dynamo", "Dynamo Core"),
```

Comment:
are these strings possibly already defined somewhere?

Reply:
These probably should be defined as constants somewhere as they are hardcoded in several places throughout the codebase. I'm not doing it here as it could clutter this PR.